packages/lesswrong/lib/collections/users/schema.ts

```diff
@@ -153,6 +153,18 @@ const schema: SchemaType<DbUser> = {
     },
     group: formGroups.default,
   },
+  /**
+   Used for tracking changes of displayName
+   */
+  previousDisplayName: {
+    type: String,
+    optional: true,
+    canUpdate: ['sunshineRegiment', 'admins'],
```

Comment:
I think the security should also allow userOwns. I guess we don't actually check security during callbacks so this is not causing an issue, but seems good to do in principle.

Reply:
You're saying that line 162 should have userOwns? That would make it appear on the form and allow a user to change their previousDisplayName, which isn't something we want.

Reply:
You can just `hidden: true` it

Reply:
But I want admins to be able to see it.